src/components/RightSidebar/Description/Description.vue

```diff
@@ -0,0 +1,324 @@
+			v-model="descriptionText"
+			class="description__contenteditable"
+			:auto-complete="()=>{}"
+			:maxlength="maxLength"
+			:contenteditable="editing && !loading"
+			:placeholder="placeholder"
+			@submit="handleSubmitDescription"
+			@keydown.esc="handleCancelEditing" />
+		<template v-if="!loading">
+			<template v-if="editing">
+				<button
+					class="description__button"
+					:aria-label="t('spreed','Cancel editing description')"
+					@click="handleCancelEditing">
+					<Close decorative
+						:size="20" />
+				</button>
+				<button
+					class="description__button primary"
+					:aria-label="t('spreed','Submit conversation description')"
+					:disabled="!canSubmit"
+					@click="handleSubmitDescription">
+					<Check decorative
+						:size="20" />
+				</button>
+				<div v-if="showCountDown"
+					v-tooltip.auto="countDownWarningText"
+					class="counter"
+					tabindex="0"
+					aria-label="countDownWarningText">
+					<span>{{ charactersCountDown }}</span>
+				</div>
+			</template>
+			<button v-if="!editing"
+				class="description__button"
+				:aria-label="t('spreed','Edit conversation description')"
+				@click="handleEditDescription">
+				<Pencil decorative
+					:size="20" />
+			</button>
+		</template>
+		<div v-if="loading" class="icon-loading-small spinner" />
+	</div>
+</template>
+
+<script>
+import Pencil from 'vue-material-design-icons/Pencil'
+import Check from 'vue-material-design-icons/Check'
+import Close from 'vue-material-design-icons/Close'
+import RichContentEditable from '@nextcloud/vue/dist/Components/RichContenteditable'
+import Tooltip from '@nextcloud/vue/dist/Directives/Tooltip'
+
+export default {
+	name: 'Description',
+	components: {
+		Pencil,
+		Check,
+		Close,
+		RichContentEditable,
+	},
+
+	directives: {
+		Tooltip,
+	},
+
+	props: {
+		/**
+		 * The description (An editable paragraph just above the sidebar tabs)
+		 */
+		descriptionTitle: {
+			type: String,
+			default: 'Description',
+		},
+
+		/**
+		 * A paragraph below the title.
+		 */
+		description: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Shows or hides the editing buttons.
+		 */
+		editable: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Toggles the description editing state on and off.
+		 */
+		editing: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Placeholder for the contenteditable element.
+		 */
+		placeholder: {
+			type: String,
+			default: '',
+		},
+
+		/**
+		 * Toggles the loading state on and off.
+		 */
+		loading: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Maximum description length in characters
+		 */
+		maxLength: {
+			type: Number,
+			default: 250,
+		},
+	},
+
+	data() {
+		return {
+			descriptionText: '',
+		}
+	},
+
+	computed: {
+		contenteditable() {
+			return this.$refs.contenteditable.$refs.contenteditable
+		},
+
+		canSubmit() {
+			return this.charactersCount <= 250 && this.descriptionText !== this.description
```

Comment:
```suggestion
			return this.charactersCount <= this.maxLength && this.descriptionText !== this.description
```

Also as per backend PR I think 250 is very short.